package.json

```diff
@@ -33,42 +33,38 @@
     "react-markdown": "^5.0.3",
     "react-router-dom": "^5.2.0",
     "styled-components": "^3.2",
-    "throttle-debounce": "^3.0.1",
     "ts-is-present": "^1.2.1"
   },
   "devDependencies": {
-    "@babel/core": "^7.12.10",
-    "@babel/plugin-transform-runtime": "^7.12.10",
-    "@babel/preset-typescript": "^7.12.7",
-    "@compiled/babel-plugin": "^0.6.0",
-    "@storybook/addon-actions": "^6.1.15",
-    "@storybook/addon-essentials": "^6.1.15",
-    "@storybook/addon-links": "^6.1.15",
-    "@storybook/react": "^6.1.15",
+    "@babel/core": "^7.22.5",
+    "@babel/plugin-transform-runtime": "^7.22.5",
+    "@babel/preset-env": "^7.22.5",
+    "@babel/preset-react": "^7.22.5",
+    "@babel/preset-typescript": "^7.22.5",
+    "@compiled/babel-plugin": "^0.19.1",
+    "@storybook/addon-actions": "^7.0.22",
+    "@storybook/addon-essentials": "^7.0.22",
+    "@storybook/addon-links": "^7.0.22",
+    "@storybook/cli": "^7.0.22",
+    "@storybook/react-webpack5": "^7.0.22",
     "@types/js-yaml": "^4.0.0",
     "@types/react": "^16.8",
     "@types/react-copy-to-clipboard": "^5.0.0",
     "@types/react-dom": "^16.8",
     "@types/react-router-dom": "^5.1.7",
-    "@types/throttle-debounce": "^2.1.0",
     "babel-loader": "^8.2.2",
     "babel-plugin-transform-class-properties": "^6.24.1",
     "csp-html-webpack-plugin": "^5.1.0",
     "css-loader": "^5.0.1",
-    "favicons": "^6.2.0",
-    "favicons-webpack-plugin": "^5.0.0-alpha.6",
-    "html-webpack-plugin": "^5.0.0-beta.4",
-    "http-server-spa": "^1.3.0",
+    "favicons": "^7.1.3",
+    "favicons-webpack-plugin": "^6.0.0",
+    "html-webpack-plugin": "^5.5.3",
     "json-schema-to-typescript": "^10.1.2",
-    "npm-run-all": "^4.1.5",
```

Comment:
@slatlasdev I was using that for the package.json scripts. Can you add it back please?

Reply:
Sure thing. 
Although if there's common sets of scripts you run together, it's often a good idea to combine them in the `package.json` for maintainability and to share with others, eg:
```json
"deploy": "npm run build-prod && npm run deploy"
```